build/package/Microsoft.DotNet.Cli.Installer.PKG.targets

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    
+    <UsingTask TaskName="PkgTargets" AssemblyFile="$(CLIBuildDll)" />
+
+    <Target Name="GeneratePkgs" >
```

Comment:
Ok, if this is going to have more stuff in it, then I'm fine with the separate file.